docs/developer-experience/artificial-intelligence.md

```diff
@@ -0,0 +1,63 @@
+# Artificial Intelligence
+
+There are a number of AI tools that can improve the developer experience. This article will discuss tooling that is available as well as advice on when it might be appropriate to use such tooling.
+
+## GitHub Copilot
+
+The current version of GitHub Copilot can provide code completion in Visual Studio Code. It is a VS Code extension that can be installed from the VS Code Marketplace. It is currently in beta and requires a GitHub account to use. It is free to use during the beta period. It is currently available for VS Code only, but there are plans to support other IDEs in the future. It is currently available for the following languages: JavaScript, TypeScript, Ruby, Go, Python, C#, C++, and Java. You can find more information about GitHub Copilot [here](https://copilot.github.com/).
+
+Developers on our team have found the following use-cases for GitHub Copilot:
+
+- __Write Documentation__. For example, the above paragraph was written using Copilot.
+- __Write Unit Tests__. Given that setup and assertions are often consistent across unit tests, Copilot tends to be very accurate.
+- __Unblock__. It is often hard start writing when staring at a blank page, Copilot can fill the space with something that may or may not be what you ultimately want to do, but it can help get you in the right headspace.
+
+If you want Copilot to write something useful for you, try writing a comment that describes what your code is going to do - it can often take it from there.
+
+## GitHub Copilot Labs
+
+Copilot has extension that offers additional features that are not yet ready for prime-time. You can install it from the VS Code Marketplace. These features include:
```

Comment:
```suggestion
Copilot has an extension that offers additional features that are not yet ready for prime-time. You can install it from the VS Code Marketplace. These features include:
```

Would be good to point to the extension page

Reply:
done.